src/main/java/org/elasticsearch/action/admin/cluster/health/ClusterHealthResponse.java

```diff
@@ -57,6 +58,8 @@
     int initializingShards = 0;
     int unassignedShards = 0;
     int numberOfPendingTasks = 0;
+    int numberOfFetchShardStarted = 0;
```

Comment:
I really can't tell what these are is this the total or the inflight? what did this fetch teh entire shard or only some info?